moderatorguide.md

```diff
@@ -45,20 +47,18 @@ In general, mediating questions works like this:
 * In the event of a prolonged discussion, you may get the talk back on track by suggesting that the conversation continue separately after the seminar.
```

Comment:
```suggestion
* In the event of a prolonged discussion, you may get the talk back on track by suggesting that the conversation continue separately after the seminar.
```
* In the event of a prolonged discussion, get the talk back on track and suggest a private discussion after the seminar. 

Same as @xabomon but with a comma